src/pages/AboutPage/AboutPage.js

```diff
@@ -1,8 +1,52 @@
 // AboutPage.js
 import React from "react";
+import { Card, CardMedia, Typography, Divider } from '@mui/material';
+import About from './Images/About.png';
```

Comment:
```suggestion
import About from '../Images/About.png';
```

Pretty sure you meant to go back a directory with `../` first. As a reminder, `./` accesses the current directory (in this case the one that AboutPage.js is in).